WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -797,6 +811,65 @@ extension ProductSelectorViewModel {
     }
 }
 
+// MARK: - Tracking
+private extension ProductSelectorViewModel {
+    func trackConfirmButtonTapped(with productsCount: Int) {
```

Comment:
Very non-blocker, just for consistency:
```suggestion
    func trackConfirmButtonTapped(with productCount: Int) {
```
Then on line 819 would be:
```diff
- analytics.track(event: WooAnalyticsEvent.Orders.orderCreationProductSelectorConfirmButtonTapped(productCount: productsCount,
+ analytics.track(event: WooAnalyticsEvent.Orders.orderCreationProductSelectorConfirmButtonTapped(productCount: productCount,
```

Reply:
Yeah, done in https://github.com/woocommerce/woocommerce-ios/pull/9586